### PR TITLE
merge: backend/dev -> main

### PR DESCRIPTION
**Pull request by Ximaz**
These commits solves a few bugs, from documentation to webhook error handling. There is still some work todo, but it may be good that you already have these changes, mostly for documentation sake.

### CODE REVIEW

File: backend/src/area/area.service.ts [hunk 1]
```diff
@@ -358,4 +356,22 @@ export class AreaService {
 
         return this.prismaAreaToArea(updated);
     }
+
+    async delete(userId: User["id"], areaId: Area["id"]) {
+        const area = await this._findUnique(areaId, userId);
+        console.log(area);
+        const task = await this.getAreaTask(area);
+        console.log(task);
```
Comment (Clement-Z4RM) [inline backend/src/area/area.service.ts:364]: You omitted `console.log`s

**Review by Clement-Z4RM — DISMISSED** [review]

File: backend/src/area/area.service.ts (hunk 1 above)
Comment (Ximaz) [reply backend/src/area/area.service.ts:364]: YOu're a 100% right, I will fix that.

**Review by Clement-Z4RM — APPROVED** [review]

**Comment by Ximaz** [comment]
Just to let you know : those fixes include the HTTP response code issue that you spotted, so it will have to be fixed on the frontend side too.

**Comment by Clement-Z4RM** [comment]
> Just to let you know : those fixes include the HTTP response code issue that you spotted, so it will have to be fixed on the frontend side too.

Are you sure about this? I don't see the fix.

**Comment by Ximaz** [comment]
> > Just to let you know : those fixes include the HTTP response code issue that you spotted, so it will have to be fixed on the frontend side too.
> 
> Are you sure about this? I don't see the fix.

Addressed here : https://github.com/zowks/B-DEV-500-area/pull/70

**Pull request merged**